lib/galaxy/webapps/base/api.py

```diff
@@ -237,16 +239,21 @@ def add_request_id_middleware(app: FastAPI):
 
 
 def include_all_package_routers(app: FastAPI, package_name: str):
+    responses: typing.Dict[typing.Union[int, str], typing.Dict[str, typing.Any]] = {
+        422: {
+            "description": "Validation error",
+            "model": MessageExceptionModel,
+        },
+    }
+    for code, error in error_codes_by_int_code.items():
+        code_int = int(str(code)[:3])
+        if code_int in [0, 204] or code_int in responses:
+            continue
+        responses[code_int] = {
```

Comment:
Do you mean using `4XX` and  `5XX` instead of all the code errors in docs?
It returns this error while updating client API schema: `SyntaxError: An identifier or keyword cannot immediately follow a numeric literal.`

Reply:
I meant the string "4XX" and "5XX", as listed in the swagger docs

Reply:
I think this is a bug in the version of `openapi-typescript` we are using:

https://github.com/galaxyproject/galaxy/actions/runs/9853821142/job/27205208465?pr=18093#step:10:189

Updating it to version `6.7.6` generates the correct typescript schema, but it also changes (probably fixes) some of the types that were generated.

For example: 
`Record<string, never>` -> `unknown` which I think makes more sense and they probably fixed it.

I can try to open a new PR for bumping the `openapi-typescript` version and fixing the new type issues in the client.

Reply:
@davelopez I can pop just that upgrade out of https://github.com/dannon/galaxy/tree/openapi-typescript-fetch-upgrade if we want, following up on the fetcher swapover later?

Reply:
Sorry I missed this earlier :sweat_smile: 
I have a PR just bumping the version to 6.7.6 and fixing the issues in the client. But I agree, we should follow up and try to swap the fetcher later.

Reply:
No worries, sounds good!